composer/trainer/trainer.py

```diff
@@ -770,8 +782,10 @@ def __init__(
         loggers: Optional[Union[LoggerDestination, Sequence[LoggerDestination]]] = None,
         run_name: Optional[str] = None,
         progress_bar: bool = True,
-        log_to_console: Optional[bool] = None,
+        log_to_console: bool = False,
         console_stream: Union[str, TextIO] = 'stderr',
+        console_log_interval: Union[int, str, Time] = 1,
```

Comment:
nit: not sure about having a unitless number be an option here. Would refer to force the convention we use for time strings everywhere else

Reply:
`eval_interval` also has this convention of unitless, but you're right it is confusing

Reply:
fixed